### PR TITLE
clean up email submission ui

### LINKED ISSUE #2153: Missing string on newsletter sign up success pop up
[Affected versions]:
- Firefox 51.0.1 and above

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- Have a valid email address that is not signed up on the Test Pilot Newsletter. 

[Steps to reproduce]:
1. Open Firefox browser with the profile from prerequisites and navigate to https://testpilot.firefox.com page.
2. Choose to "Install the Test Pilot Add-on".
3. After the "Welcome" pop up is displayed, enter a valid email address in the textbox for Newsletter sign up.
4. Click on the "Sign me up" blue button.
5. Observe the elements displayed on the confirmation pop up.

[Expected result]:
- The confirmation pop up is correctly displayed and contains all the designed elements.

[Actual result]:
- The confirmation pop up is displayed but misses the "When you get a chance, head over to your inbox to verify your email address." string before the "Thank you!" one.

[Notes]:
- It seams that the string was lost in the process. We should make sure the string is also translated before adding it back.
- Here is a screenshot of the current pop up:
newsletter modal missing

### PR DESCRIPTION
**Pull request by johngruen**
This started as a fix for #2153 but i tried to clean up some cruft along the way. Unfortunately i'm failing two tests now, which i'm not sure about. Could use a look!

-fixes #2153

### CODE REVIEW

**Comment by fzzzy** [comment]
I think this needs to be rebased, @johngruen

**Comment by ghost** [comment]
r+ for the new dates.  /cc @flodolo for the new string

File: locales/en-US/app.ftl [hunk 1]
```diff
@@ -75,8 +75,8 @@ notFoundHeader = Four Oh Four!
 emailOptInDialogTitle = Welcome to Test Pilot!
 emailOptInMessage = Find out about new experiments and see test results for experiments you've tried.
 emailOptInConfirmationTitle = Email Sent
-emailOptInSuccessMessage2 = Thank you!
 emailOptInConfirmationClose = On to the experiments...
+emailOptInDialogErrorTitle = On no!
```
Comment (flodolo) [inline locales/en-US/app.ftl]: On no? Should this be "O**h** no"?

Comment (johngruen) [reply locales/en-US/app.ftl]: nice catch!

**Pull request merged**